pages/en/deploying/graph-hosted-service-sunsetting-faqs.mdx

```diff
@@ -58,9 +58,9 @@ Core developers are planning to add as many chains to the decentralized network
 
 We would love to hear from you! Please tag The Graph on Twitter or comment in [Discord](https://thegraph.com/discord) to let the ecosystem know the chain(s) you would like The Graph ecosystem to prioritize adding to The Graph Network next.
 
-### 9. Will the decentralized network have the same core functionalities as the hosted service??
+### 9. Will the decentralized network have the same core functionalities as the hosted service?
 
-Yes, the decentralized network provides all core functionalities of the hosted service while being significantly more robust, secure, and reliable. Instead of just one indexing service, The Graph Network is powered by 169 (and counting) independently operated and geographically distributed Indexers.
+Yes, the decentralized network provides all core functionalities of the hosted service while being significantly more robust, secure, and reliable. Instead of just one indexing service, The Graph Network is powered by 201 (and counting) independently operated and geographically distributed Indexers.
```

Comment:
We're actually up to [224 Indexers](https://thegraph.com/explorer/participants/indexers)!